.travis.yml

```diff
@@ -0,0 +1,38 @@
+language: python
```

Comment:
I don't have enough permissions to join the project to travis. Meaning that I can't test this and I can't get the link (well, its craftable, but I don't want to add it without a test).

If you can register this project on travis, I'll do it for you.